.github/workflows/build.yaml

```diff
@@ -206,99 +206,22 @@ jobs:
 
     runs-on: ubuntu-latest
 
+    if: startsWith(github.ref, 'refs/tags/')
+
     steps:
     - uses: actions/checkout@v4
 
     - name: Download all artifacts
       uses: actions/download-artifact@v4
 
-    - name: Setup ssh-agent
-      env:
-        SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
-      if: ${{ env.SSH_PRIVATE_KEY != '' }}
-      uses: webfactory/ssh-agent@v0.9.0
-      with:
-        ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
-
-    - name: Import GPG key
-      id: import_gpg
-      env:
-        GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
-        GPG_PRIVATE_KEY_PASSPHRASE: ${{ secrets.GPG_PRIVATE_KEY_PASSPHRASE }}
-      if: ${{ env.GPG_PRIVATE_KEY != '' && env.GPG_PRIVATE_KEY_PASSPHRASE != '' }}
-      uses: crazy-max/ghaction-import-gpg@v6
-      with:
-        gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
-        passphrase: ${{ secrets.GPG_PRIVATE_KEY_PASSPHRASE }}
-
-    - name: Configure ssh known hosts for gitlab.com
-      env:
-        SSH_PRIVATE_KEY: ${{ secrets.SSH_PRIVATE_KEY }}
-      if: ${{ env.SSH_PRIVATE_KEY != '' }}
-      run: cat gitlab_known_hosts >> ~/.ssh/known_hosts
-
-    - name: Configure git user
+    - name: Rename artifacts
       run: |
-        git config --global user.email 'openmw-deps-build@users.noreply.github.com'
-        git config --global user.name 'openmw-deps-build'
+        mv vcpkg-x64-*-${{ github.sha }}/vcpkg-x64-windows-2019-${{ github.sha }}.7z windows-2019.7z
+        mv vcpkg-x64-*-${{ github.sha }}/vcpkg-x64-windows-2022-${{ github.sha }}.7z windows-2022.7z
+        mv vcpkg-x64-*-${{ github.sha }}/vcpkg-x64-windows-2019-pdb-${{ github.sha }}.7z windows-2019-pdb.7z
+        mv vcpkg-x64-*-${{ github.sha }}/vcpkg-x64-windows-2022-pdb-${{ github.sha }}.7z windows-2022-pdb.7z
```

Comment:
I think the renaming is bad because we're still going to expect people to have multiple versions downloaded side by side in their `openmw/deps` directory, and we need the CI script to be able to tell whether it's got to download a new one and which one it wants to extract.

Reply:
It has the benefit of not requiring two variables (hash and tag) to download the relevant files. We can just put the tag in the downloaded files or download them to a folder named for the tag.

Reply:
If you want just one variable, put the tag in the filename instead of the commit hash. Having directories with a single file in them is silly, and needing to rename stuff when we download it is an avoidable hassle.

Reply:
That seems fine, yeah.

Though pushing the url to the other repo would also resolve the issue.

Reply:
The tag's more readable than the commit hash and can still be turned back into the hash, so it's probably the better thing to have in the filename.